api/v1/sriovnetworknodestate_types.go

```diff
@@ -64,6 +66,7 @@ type InterfaceExt struct {
 	NumVfs            int               `json:"numVfs,omitempty"`
 	LinkSpeed         string            `json:"linkSpeed,omitempty"`
 	LinkType          string            `json:"linkType,omitempty"`
+	LinkFlags         net.Flags         `json:"linkFlags,omitempty"`
```

Comment:
I don't know if I will like to expose this into the interfaceExt struct this is a uint number that doesn't mean much
maybe we have something like LinkOprState string - up/down